lib/util.js

```diff
@@ -128,6 +128,28 @@ exports.chooseFormat = (formats, options) => {
       format = formats[formats.length - 1];
       break;
 
+    case 'highestaudio':
+      var highest = formats[0];
+      for (var format of formats) {
+        if (!highest.audioBitrate || (format.audioBitrate
```

Comment:
No need to check for `format.audioBitrate` since if it gets to this point, then we can safely assume it's truthy